templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml

```diff
@@ -47,28 +49,38 @@ contents:
           set -euxo pipefail
 
           {{if .Images.clusterEtcdOperatorImageKey}}
-            COMMAND="cluster-etcd-operator mount"
+            SERVER_COMMAND="cluster-etcd-operator mount"
+            PEER_COMMAND="cluster-etcd-operator mount"
+            METRIC_COMMAND="cluster-etcd-operator mount"
           {{else}}
-            COMMAND="kube-client-agent request"
+            COMMAND="kube-client-agent request \
+                --dnsnames={{etcdServerCertDNSNames .}} \
+                --commonname=system:etcd-server:${ETCD_DNS_NAME} \"
```

Comment:
--commonname is a dupe here right?

Reply:
nice catch, thanks

Reply:
kubeconfig flag was missing, added, PTAL